lib/crypto.js

```diff
@@ -38,6 +39,16 @@ export default class Crypto extends CryptoBase {
     return 'Sodium'
   }
 
+  /**
+   * Gets the signature version currently supported by this crypto object.
+   *
+   * @return {string} The current signature version in UUID-v5 format.
```

Comment:
is it ok that this is uuid5 and the rest uses uuid4?

Reply:
yes -- UUIDv5 is deterministic based on an input and a UUIDv4 namespace. This UUIDv5 represents the signing protocol in sodium mode. Ethans spec details this here: https://github.com/tozny/internal-docs/blob/master/tozny-platform/notes/tozny-field-signing.md#tozny-field-signing-protocol-with-blake2b-and-ed25519